infrastructure/terraform/module-ecs-services/security-groups.tf

```diff
@@ -1,19 +1,19 @@
 resource "aws_security_group" "internal-service-sg" {
   description = "Security group for internal service albs"
-  vpc_id      = "${var.vpc_id}"
+  vpc_id      = var.vpc_id
 
   ingress {
     from_port   = 80
     to_port     = 80
     protocol    = "tcp"
-    cidr_blocks = flatten([split(",", var.internal_cidrs), split(",", var.application_cidrs)])
+    cidr_blocks = flatten([split(",", var.internal_cidrs), split(",", var.application_cidrs), var.mgmt-eu-west-1_cidrs])
```

Comment:
As mentioned, do the aggregation externally and then just pass in the values

Reply:
Done. See comments: https://github.com/companieshouse/test-data-generator-stack/pull/2#discussion_r386908345